### ISSUE TITLE
Failed to activate the text-pastry package

### ISSUE THREAD

**Issue by steveostudios** [seveibar/text-pastry#16]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.0.15
**System**: Mac OS X 10.10.5
**Thrown From**: [text-pastry](https://github.com/seveibar/text-pastry) package, v0.4.3

### Stack Trace

Failed to activate the text-pastry package

```
At Cannot read property 'prototype' of undefined

TypeError: Cannot read property 'prototype' of undefined
  at __extends (/Users/steve/.atom/packages/text-pastry/lib/text-pastry-view.coffee:1:1)
  at /Users/steve/.atom/packages/text-pastry/lib/text-pastry-view.coffee:9:5
  at Object.<anonymous> (/Users/steve/.atom/packages/text-pastry/lib/text-pastry-view.coffee:94:5)
  at Object.<anonymous> (/Users/steve/.atom/packages/text-pastry/lib/text-pastry-view.coffee:1:1)
  at Module._compile (module.js:428:26)
  at Object.defineProperty.value [as .coffee] (/Applications/Atom.app/Contents/Resources/app.asar/src/compile-cache.js:169:21)
  at Module.load (module.js:353:32)
  at Function.Module._load (module.js:308:12)
  at Module.require (module.js:363:17)
  at require (module.js:382:17)
  at Object.<anonymous> (/Users/steve/.atom/packages/text-pastry/lib/text-pastry.coffee:1:18)
  at Object.<anonymous> (/Users/steve/.atom/packages/text-pastry/lib/text-pastry.coffee:1:1)
  at Module._compile (module.js:428:26)
  at Object.defineProperty.value [as .coffee] (/Applications/Atom.app/Contents/Resources/app.asar/src/compile-cache.js:169:21)
  at Module.load (module.js:353:32)
  at Function.Module._load (module.js:308:12)
  at Module.require (module.js:363:17)
  at require (module.js:382:17)
  at Package.module.exports.Package.requireMainModule (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:761:34)
  at Package.module.exports.Package.activateConfig (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:263:14)
  at Package.module.exports.Package.activateNow (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:241:14)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:838:25
  at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:82:11)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:226:20)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at CommandPaletteView.module.exports.CommandPaletteView.confirmed (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/command-palette/lib/command-palette-view.js:159:32)
  at CommandPaletteView.module.exports.SelectListView.confirmSelection (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:338:21)
  at space-pen-div.atom.commands.add.core:confirm (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:109:19)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:245:29)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:524:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:347:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:184:20)

```

### Commands

```
     -4:02.1.0 core:paste (atom-text-editor.editor.is-focused)
     -4:00.1.0 emmet:insert-formatted-line-break-only (atom-text-editor.editor.is-focused)
     -4:00.1.0 editor:newline (atom-text-editor.editor.is-focused)
     -3:59.7.0 core:paste (atom-text-editor.editor.is-focused)
 15x -3:50.6.0 find-and-replace:select-next (atom-text-editor.editor.is-focused)
  3x -3:47 core:move-right (atom-text-editor.editor.is-focused)
  2x -3:46.1.0 core:backspace (atom-text-editor.editor.is-focused)
     -1:13.1.0 command-palette:toggle (atom-text-editor.editor.is-focused)
  9x -1:10 core:backspace (atom-text-editor.editor.mini.is-focused)
  7x -1:06.2.0 core:move-down (atom-text-editor.editor.mini.is-focused)
     -1:01.1.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -1:01.1.0 settings-view:install-packages-and-themes (atom-text-editor.editor)
     -0:56.1.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:07.8.0 command-palette:toggle (atom-text-editor.editor.is-focused)
     -0:01.3.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:01.3.0 text-pastry:paste-1-to-x (atom-text-editor.editor)
```

### Config

```json
{
  "core": {}
}
```

### Installed Packages

```coffee
# User
atom-beautify, v0.28.12
auto-detect-indentation, v0.4.2
autoclose-html, v0.19.0
emmet, v2.3.12
highlight-selected, v0.10.1
linter, v1.6.0
linter-bootlint, v0.0.6
linter-csslint, v1.0.5
linter-htmlhint, v0.1.1
linter-jshint, v1.2.0
minimap, v4.13.3
minimap-highlight-selected, v4.3.0
minimap-linter, v1.0.0
open-terminal-here, v2.1.0
text-pastry, v0.4.3
wordcount, v2.3.0

# Dev
No dev packages
```

**Comment by kaushikgopal** [comment]
+1 doesn't allow me to use the package.

**Comment by Twizzledrizzle** [comment]
Same here, have any of you found a way around this? This package on sublime is big time saver for me

**Comment by alexbet** [comment]
+1 Loved the package on Sublime, but this one doesn't work at all. Hope the issue get's resolved.

**Comment by seveibar** [comment]
fixed with #17, thanks for reporting!
